Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY vendor/ vendor/
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go
 
-FROM registry.ci.openshift.org/ocp/4.10:base
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 as builder
```

Comment:
This is the final runtime image, shouldnt it be debian to go on the same OS than the builder?

Reply:
This was a type, it is fixed.
Debian currently isn't working, I am not sure why.
Trying now with `registry.access.redhat.com/ubi8/ubi`